src/Elliptic/Systems/Poisson/Equations.cpp

```diff
@@ -0,0 +1,161 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Elliptic/Systems/Poisson/Equations.hpp"
+
+#include <array>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Variables.hpp"  // IWYU pragma: keep
+#include "Elliptic/Systems/Poisson/MixedSystem.hpp"
+#include "Elliptic/Systems/Poisson/Tags.hpp"  // IWYU pragma: keep
+#include "NumericalAlgorithms/LinearOperators/Divergence.hpp"  // IWYU pragma: keep
+#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp"  // IWYU pragma: keep
+#include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "Utilities/TMPL.hpp"
+// IWYU pragma: no_forward_declare Tags::deriv
+// IWYU pragma: no_forward_declare Tags::div
+// IWYU pragma: no_forward_declare Variables
+// IWYU pragma: no_forward_declare Tensor
+
+namespace Poisson {
+
+template <size_t Dim>
+void ComputeMixedVolume<Dim>::apply(
+    const gsl::not_null<Scalar<DataVector>*> volume_for_field_source,
+    const gsl::not_null<tnsr::I<DataVector, Dim, Frame::Inertial>*>
+        volume_for_auxiliary_field_source,
+    const tnsr::i<DataVector, Dim, Frame::Inertial>& grad_field,
+    const tnsr::I<DataVector, Dim, Frame::Inertial>& auxiliary_field,
+    const Mesh<Dim>& mesh,
+    const InverseJacobian<DataVector, Dim, Frame::Logical, Frame::Inertial>&
+        inverse_jacobian) noexcept {
+  auto div_vars = make_with_value<Variables<tmpl::list<AuxiliaryField<Dim>>>>(
+      auxiliary_field, 0.);
+  get<AuxiliaryField<Dim>>(div_vars) = auxiliary_field;
+  *volume_for_field_source = get<Tags::div<AuxiliaryField<Dim>>>(
```

Comment:
why `-1.0 * ...` rather than multiplying by `-1` after? If there's a good reason could you please add a comment in the code?